activerecord/lib/active_record/autosave_association.rb

```diff
@@ -398,7 +398,7 @@ def save_collection_association(reflection)
                 if autosave
                   saved = association.insert_record(record, false)
                 elsif !reflection.nested?
-                  association_saved = association.insert_record(record)
+                  association_saved = association.insert_record(record, association.owner.run_validations?)
```

Comment:
Isn't the current scope the owner? Would the following work as well?
`association_saved = association.insert_record(record, run_validations?)`